### PR TITLE
Equilibration with water-gas ratio vs depth (RVWVD) table

### PR DESCRIPTION
**Pull request by plgbrts**

### CODE REVIEW

**Comment by plgbrts** [comment]
This PR is related to https://github.com/OPM/opm-common/pull/3107

**Comment by plgbrts** [comment]
jenkins build this opm-common=3107

**Comment by plgbrts** [comment]
jenkins build this opm-common=3107 please

**Comment by plgbrts** [comment]
jenkins build this opm-common=3107 please

**Comment by plgbrts** [comment]
jenkins build this opm-common=3107 please

File: ebos/equil/initstateequil.hh [hunk 1]
```diff
@@ -128,12 +128,14 @@ private:
                    const double press) const;
 };
 
-template <class FluidSystem, class RV>
+//TODO:add argument rvw
+template <class FluidSystem, class RV, class RVW>
```
Comment (bska) [inline ebos/equil/initstateequil.hh:132]: Is this comment left over from earlier development or is there still something missing here?

**Review by bska — APPROVED** [review]
This looks good to me.  We may want to look into refactoring the implementation of the gas density function, but that's a topic for future work.  Other than a possibly misplaced/outdated comment I'm ready to merge this.

File: ebos/equil/initstateequil.hh (hunk 1 above)
Comment (plgbrts) [reply ebos/equil/initstateequil.hh]: This is an obsolete comment. I removed it.

**Comment by bska** [comment]
Thank you for the updates.  I'll merge this into master now.

**Pull request merged**